quarkus/tests/integration/src/main/java/org/keycloak/it/junit5/extension/CLITestExtension.java

```diff
@@ -72,6 +73,8 @@ public void beforeEach(ExtensionContext context) throws Exception {
             }
         } else {
             configureProfile(context);
+            // WORKAROUND: this intercepts the output when actually starting the server.
+            QuarkusConsole.installRedirects();
```

Comment:
This workaround is fixed here:

https://github.com/quarkusio/quarkus/pull/22455

Reply:
Looks like we are in the same situation. Do we really need this workaround to make tests stable? I understand it might help when running tests locally out of order, etc.

But I would say we can just wait for the fix from Quarkus.

Reply:
I have no issue in merging with this, but I would prefer to introduce this change only if needed.

Reply:
> Do we really need this workaround to make tests stable?

We do need this workaround for the JVM tests to pass at all ...

Reply:
We do not, at least I can't see any failure looking at the last runs. Neither locally.